.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,56 @@
-#### Submission Checklist
+Thanks for submitting a pull request! Please remove this text when submitting. We expect everything in the checklist to be completed before the pull request is merged. If any is missing, the pull request may be closed until it's ready. Please help guide the reviewer through your code (see [Code Review Guidelines](https://github.com/stan-dev/math/wiki/Developer-Doc#code-review-guidelines)).
 
-- [ ] Run unit tests: `./runTests.py test/unit`
-- [ ] Run cpplint: `make cpplint`
-- [ ] Declare copyright holder and open-source license: see below
+## Checklist
 
-#### Summary:
+- [ ] Math issue #(issue number)
 
-#### Intended Effect:
+- [ ] Copyright holder: (fill in copyright holder information)
 
-#### How to Verify:
+    The copyright holder is typically you or your assignee, such as a university or company. By submitting this pull request, the copyright holder is agreeing to the license the submitted work under the following licenses:
+      - Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
+      - Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
 
-#### Side Effects:
+- [ ] the code base is stable
 
-#### Documentation:
+    - all unit tests pass
+    - continuous integration passes
 
-#### Copyright and Licensing
+- [ ] the changes are maintainable
 
-Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):
+    - the code design is standard C++ or there's a good reason it's not
```

Comment:
What does this mean, standard C++? Maybe link to our guidelines wiki (the one where we say how we're different from Google) or something? Every company uses a different subset of C++ from what I can tell.

Reply:
I'll change that to "idiomatic" before we merge. We can make adjustments to the pull request template if it seems too long / cumbersome.